Added "Why Filecoin?" page (#24)
filecoin-project/filecoin-docs: docs/concepts/why-filecoin.md

---
title: Why Filecoin?
---

# Why Filecoin?

## Open market

In Filecoin, file storage and retrieval is negotiated in open markets. Anybody can join the Filecoin network without needing permission. Running a miner only requires an internet connection and spare disk space. By lowering the barriers to entry, Filecoin enables a thriving ecosystem of many independent storage providers.

## Competitive prices

Prices for storage and retrieval are determined by supply and demand, not corporate pricing departments. Filecoin makes reliable storage available at hypercompetitive prices. Miners compete based on their storage, reliability and speed rather than through marketing or attempting to lock users in.

## Reliable storage

> Because storage is paid for, Filecoin provides a viable economic reason for files to stay available over time. Files are stored on computers that are reliable and well-connected to the internet. Although Filecoin is a peer-to-peer network, it does not rely solely on an ephemeral swarm of peers for files to stay online.

Reviewer: "an ephemeral swarm of peers" is a phrase folks can't understand without a fair amount of pre-existing knowledge of P2P. Do you mean that you don't have to have a ton of people online simultaneously?

Author: Yes I agree this sentence was confusing and wasn’t adding much useful information so I removed it.

## Self-healing

The Filecoin network continually verifies that files are stored correctly. The Filecoin blockchain has a built-in self-healing process where faulty miners are detected and their files are redistributed to reliable miners.

## Verifiable traces

In the process of self-healing, Filecoin generates verifiable traces that files have been stored correctly over time. Clients can efficiently scan these traces to confirm that their files have been stored correctly, even if the client was offline at the time. Any observer can check any miner’s track record and will notice if the miner has been faulty or offline in the past.

> ## Reputation not marketing

Reviewer: ```suggestion
## Reputation, not marketing
```

In Filecoin, storage providers prove their reliability through their track record published on the blockchain, not through marketing claims published by the provider themselves. Users don’t need to rely on status pages or self-reported statistics from storage providers.

## Choice of tradeoffs

> Users get to choose their own tradeoffs between cost, redundancy and speed. Users are not limited to a set list of data centres offered by their provider, but can choose to store their files on any miner participating in Filecoin.

Reviewer: Are we standardizing on American English spelling? 

Do you store something "on" a miner (the computer) or "with" a miner (the person)? 

```suggestion
Users get to choose their own tradeoffs between cost, redundancy and speed. Users are not limited to a set list of data centers offered by their provider, but can choose to store their files on any miner participating in Filecoin.
```

Reviewer 2: Yes, let's standardize on American English (another inheritance from [IPFS](https://deploy-preview-24--filecoin-docs.netlify.app/community/contribute/grammar-formatting-and-style/#grammar-and-spelling)).

## Puts empty storage to use

> Filecoin puts disk space that would otherwise sit empty to use. Miners are incentivised to join Filecoin because they get paid for their participation, unlike other distributed networks where there is no economic incentive. Filecoin also makes it easy to join because miners self-manage their storage, earning Filecoin tokens without needing human supervision.

Reviewer: ```suggestion
Filecoin puts disk space that would otherwise sit empty to use. Miners are incentivized to join Filecoin because they get paid for their participation, unlike other distributed networks where there is no economic incentive. Filecoin also makes it easy to join because miners self-manage their storage, earning Filecoin tokens without needing human supervision.
```

## Censorship resistance

Filecoin resists censorship because there is no central provider that can be coerced into deleting files or withholding service. The network is made up of many different computers run by many different people and organizations. Faulty or malicious actors are noticed by the network and removed automatically.

## Useful blockchain

> In Filecoin, miners are rewarded for providing storage, not for performing wasteful computations. Filecoin secures its blockchain using proof of file replication and proof of storage over time. It doesn’t rely on energy intensive proof-of-work schemes like other blockchains. Miners are incentivised to amass hard drives and put them to use by storing files. Filecoin doesn’t incentivise hoarding of graphics cards or application-specific integrated circuits for the sole purpose of mining.

Reviewer: ```suggestion
In Filecoin, miners are rewarded for providing storage, not for performing wasteful computations. Filecoin secures its blockchain using proof of file replication and proof of storage over time. It doesn’t rely on energy-intensive proof-of-work schemes like other blockchains. Miners are incentivized to amass hard drives and put them to use by storing files. Filecoin doesn’t incentivize hoarding of graphics cards or application-specific integrated circuits for the sole purpose of mining.
```

## Provides storage to other blockchains

> Filecoin excels at storing large files, whereas other blockchains can typically only store very small amounts of data, very expensively. Filecoin can provide its storage to other blockchains, allowing them to store large files. In the future, mechanisms will be added to Filecoin enabling transactions on Filecoin’s blockchain to interoperate with transactions on other blockchains.

Reviewer: Part of this paragraph suggests Filecoin IS a blockchain and part suggests it USES a blockchain. I assume the latter is more accurate, and elsewhere we're referring to it as a network.

Author: Thanks! I have reworded this part to make the difference clearer.

## Content addressing

Files are referred to by the data they contain, not by fragile identifiers such as URLs. Files remain available no matter where they are hosted or who they are hosted by. When a file becomes popular it can be quickly distributed by swarms of computers instead of relying on a central computer which easily becomes overloaded by network traffic.

> When multiple users store the same file (and choose to make the file public by not encrypting it), everyone who wants to download the file benefits from Filecoin keeping it available. No matter where a file is downloaded from, downloaders can verify they have received the correct file and that it is intact.

Reviewer: This paragraph makes it sound like it's easy to download a file that's been stored on Filecoin, which isn't the impression I have from discussions about the retrieval process. @mishmosh can say better than I can whether this is a fair representation. 
```suggestion
When multiple users store the same file (and choose to make the file public by not encrypting it), everyone who wants to download the file benefits from Filecoin keeping it available. No matter where a file is downloaded from, downloaders can verify that they have received the correct file and that it is intact.
```

Author: Good call, this was probably too generous a representation. I’ve removed the paragraph.

## No lock-in

Migrating to a different storage provider is made easier because they all offer the same set of services and API. Users aren’t locked in to providers because they rely on a particular feature of the provider. In addition, files are content-addressed, which enables them to be transferred directly between miners without the user having to download and re-upload the files.

Traditional cloud storage providers lock users in by making it cheap to store files but expensive to retrieve them again. Filecoin avoids this by facilitating a retrieval market where miners compete to give users their files back as fast as possible, at the lowest possible price. 

## Open source code

The code that runs both clients and storage providers is open source. Storage providers don’t have to develop their own software for managing their infrastructure. Everyone benefits from improvements made to Filecoin’s code.

## Active community

> Filecoin has an active community of contributors to answer questions and help newcomers get started. There is an open dialog between users, developers and storage providers. If you need help, you will likely be able to reach the person who designed or built the system in question.

Reviewer: Link to community section so people can find the appropriate forums?

Author: Done!
